Revit_Engine/Revit_Engine.csproj

```
@@ -85,6 +85,10 @@
       <HintPath>C:\ProgramData\BHoM\Assemblies\Library_Engine.dll</HintPath>
       <Private>False</Private>
     </Reference>
+    <Reference Include="Physical_Engine, Version=4.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\ProgramData\BHoM\Assemblies\Physical_Engine.dll</HintPath>
```

Could you please change the path to non-conditional, i.e. _C:\ProgramData\(...)_?
```suggestion
      <HintPath>C:\ProgramData\BHoM\Assemblies\Physical_Engine.dll</HintPath>
```